src/Bits.hs

```diff
@@ -31,3 +31,8 @@ nor x y = not (or x y)
 
 xnor :: Bit -> Bit -> Bit
 xnor x y = not (xor x y)
+
+implies :: Bit -> Bit -> Bit
+implies I I = I
+implies I O = O
+implies O _ = I
```

Comment:
What do you think about making it an operator? We could then write more naturally:

```haskell
if a => b then ...
```

On the other hand, we could reserve the operator notation for the more real-world-looking implementation in terms of `NAND`, and then test against the sane-looking one that uses pattern matching.

Reply:
We didn't use operator notation for the other functions either, I think it makes sense to use either verbs or signs for all the functions.